FreeRTOS/FreeRTOS-Kernel
# Resolved Issue #164 regarding timer AutoReload data types

Fix for Issue #164 

Description
-----------
Issue #164 describes using UBaseType_t where BaseType_t should have been used.  This is in the timer creation where the reload mode of the timer must be specified.  

Test Steps
-----------
Customer detected this with a MISRA compliance check that noticed the type of pdTrue and pdFalse were BaseType_t but the API expected a UBaseType_t.

Related Issue
-----------
#164 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
